model/src/main/java/com/epam/idea/domain/User.java

```diff
@@ -1,102 +1,222 @@
 package com.epam.idea.domain;
 
+import org.hibernate.annotations.Type;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.Table;
-import javax.persistence.Id;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
-import javax.persistence.Column;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
 import javax.persistence.OneToMany;
-import javax.persistence.CascadeType;
-import java.util.List;
+import javax.persistence.PrePersist;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+import java.util.HashSet;
+import java.util.Set;
 
 @Entity
 @Table(name = "USERS")
```

Comment:
I've read [this](http://oracle-base.com/articles/misc/naming-conventions.php) article about naming conversions and it suggests that table names should be plural + I think its more readable in quarries, e.g.:
``` sql
select USER from USERS where...
```
or
``` sql
select USER from USER where...
```